packages/map/examples/default-single-state.json

```diff
@@ -1,109 +1,226 @@
 {
```

Comment:
could this go in storybook or should it be attached to the ticket? otherwise delete